.teamcity/src/main/kotlin/configurations/PerformanceTest.kt

```diff
@@ -68,20 +69,22 @@ class PerformanceTest(
                 killGradleProcessesStep(os)
                 substDirOnWindows(os)
 
-                gradleWrapper {
-                    name = "GRADLE_RUNNER"
-                    tasks = ""
-                    workingDir = os.perfTestWorkingDir
-                    gradleParams = (
-                        performanceTestCommandLine(
-                            "clean ${performanceTestTaskNames.joinToString(" ") { "$it --channel %performance.channel% ${type.extraParameters}" }}",
-                            "%performance.baselines%",
-                            extraParameters,
-                            os
-                        ) +
-                            buildToolGradleParameters() +
-                            buildScanTag("PerformanceTest")
-                        ).joinToString(separator = " ")
+                repeat(if (performanceTestBuildSpec.type == PerformanceTestType.flakinessDetection) 2 else 1) {
+                    gradleWrapper {
+                        name = "GRADLE_RUNNER${if (it != 0) "_2" else ""}"
+                        tasks = ""
+                        workingDir = os.perfTestWorkingDir
+                        gradleParams = (
+                            performanceTestCommandLine(
+                                "clean ${performanceTestTaskNames.joinToString(" ") { "$it --channel %performance.channel% ${type.extraParameters}" }}",
```

Comment:
I think we should drop the `clean` for the second run, so we don't need to rebuild nor the distribution nor the performance test project.

Reply:
done